pyproject.toml

```diff
@@ -1,3 +1,138 @@
 [build-system]
 # Minimum requirements for the build system to execute.
-requires = ["setuptools", "wheel", "setuptools_scm"]
+requires = [
+    "setuptools>=47.2.0",
+    "wheel",
+    "setuptools_scm[toml]",
```

Comment:
Why don't we need to add 'pip>=21.2.0' here as described in the commit message?

Reply:
I tried adding that and built on a Debian system shipping pip 20.3.4, but it has no effect. The old pip is used for installation.

See also https://github.com/pypa/pip/issues/4145

Reply:
I've added this information to the commit message.